### ISSUE TITLE
NVDA inconsistent in announce style changes during typing in Word

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#5894]
In Word (2016 at least), if you select Heading level 1 in the style pane, type your heading then press enter, NVDA announces the change back to Normal style.

If you press control+alt+1 to get the same Heading level 1, type your heading then press enter, NVDA does not announce the change back to Normal style.

**Comment by jcsteh** [comment]
Technical: Maybe we could call `speakTextInfo` with `REASON_ONLYCACHE` when these commands are pressed.

CC @MichaelDCurran.

**Comment by Adriani90** [comment]
@Qchristensen does it make a difference when you enable UIA in MS Word? Or is the issue still occuring anyway?

**Comment by Qchristensen** [comment]
Using the latest alpha-16793,31692c6c and Office 365 Version: 16.0.11231.20164 I get the same results as originally, regardless of whether I set NVDA to use UIA or not.

**Comment by Adriani90** [comment]
This is still reproducible with NVDA 2025.1.2 RC and MS Office 365 version 2505.
It seems in the case of alt+1,2,3 etc. there is a linefeedcharacter reported after the last character of the heading. However, it is not a linefeed character as such, it is the formating mark.

When placing the system caret after the last letter of the heading on this formating mark and press enter, the style change is reported as expected.
